gunicorn.config.py

```diff
@@ -22,6 +22,6 @@ def on_starting(server):
 """
 \x1b[0m
 """)
-    print("Server running on \x1b[4mhttp://127.0.0.1:8000\x1b[0m")
+    print("Server running on \x1b[4mhttp://0.0.0.0:8000\x1b[0m")
```

Comment:
While we're at it, `'{}:{}'.format(*server.address[0])` will actually give the correct address all the time